cmd/keytransparency-signer/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM golang
+
+ENV DB_USER=test \
+    DB_PASSWORD=zaphod \
+    DB_DATABASE=test \
+    DB_HOST=127.0.0.0:3306
+
+ENV MAP_ID=0 \
+    MAP_URL=""
+ENV LOG_ID=0 \
+    LOG_URL=localhost:8090 \
+    LOG_KEY=genfiles/trillian-log-pubkey.pem
+
+ENV SIGN_PERIOD=5s \
+    SIGN_KEY=genfiles/p256-key.pem
+
+ADD . /go/src/github.com/google/keytransparency
+WORKDIR /go/src/github.com/google/keytransparency 
+
+RUN go get -tags="mysql" ./cmd/keytransparency-signer
+
+ENTRYPOINT /go/bin/keytransparency-signer \
+	--db="${DB_USER}:${DB_PASSWORD}@tcp(${DB_HOST})/${DB_DATABASE}" \
+	--period="$SIGN_PERIOD" --key="$SIGN_KEY" \
```

Comment:
After chatting with @RJPercival (thanks again), I've decided that it's reasonable to add an additional flag for the password of the private-key/PEM-file with an option for a config-file (with the PW contained) like implemented for trillian in google/trillian/pull/620